src/pages/sdk/index.tsx

```diff
@@ -0,0 +1,122 @@
+import { theme } from '@apify-packages/ui-library';
+import Layout from '@theme/Layout';
+import React from 'react';
+import styled from 'styled-components';
+
+import Hero from '../../components/Hero/Hero';
+import SdkSection from '../../components/SdkSection/SdkSection';
+import Section from '../../components/Section/Section';
+import UiLibraryWrapper from '../../components/UiLibraryWrapper';
+
+const javascriptExample = `// Apify SDK v3 uses named exports instead of the Apify object.
```

Comment:
@baldasseva I was thinking about it and let's remove all the comments from the js SDK example (except the initial sentence from the Python example) here to keep it consistent with the Python one - there are a lot of comments explaining the migration (which happened years ago). We don't need to have it there anymore.

It'll help with the length of the example and we don't need to hardcore the limit height with the additional styling.

Reply:
<img width="771" alt="image" src="https://github.com/apify/apify-docs/assets/37815707/5f00c607-d865-4436-b238-53eb37170310">
How about this? JS is still longer, but it's acceptable